site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -64,7 +64,8 @@ scripts cannot use `tabs.query`.
 //// background.js
```

Comment:
Ugh, github doesn't let me comment on lines above this.

Can we expand this?

This example demonstrates how the background script can retrieve the currently focused tab.

->

This example demonstrates how the background script can retrieve the active tab from the currently-focused window (or most recently-focused window, if no Chrome windows are focused) - this can usually be thought of as the user's active tab.

Or similar

Reply:
Done.